modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -156,91 +156,90 @@ See xref:authentication-authorization/sso-integration.adoc#auth-sso-auth-provide
-+
 [source, properties]
 ----
 # Configure the access_token
 dbms.security.oidc.azure.config=principal=unique_name;code_challenge_method=S256;token_type_principal=access_token;token_type_authentication=access_token
 # Configure the OIDC token endpoint with the Directory (tenant) ID
 dbms.security.oidc.azure.token_endpoint=https://login.microsoftonline.com/54e85725-ed2a-49a4-a19e-11c8d29f9a0f/oauth2/v2.0/token
 # Configure the iss claim in the id token with the Directory (tenant) ID
-# Make sure you add the trailing slash (`/`) at the end of the URL, or this operation might fail.
+# Make sure you add the trailing slash (`/`) at the end of the URL or this operation might fail.
 dbms.security.oidc.azure.issuer=https://sts.windows.net/54e85725-ed2a-49a4-a19e-11c8d29f9a0f/
 # Provide the Entra ID parameters, such as client_id, response_type, scope, etc.
 dbms.security.oidc.azure.params=client_id=4376dc8b-b5af-424f-9ada-c1c1b2d416b9;response_type=code;scope=openid profile email api://4376dc8b-b5af-424f-9ada-c1c1b2d416b9/access-token
 ----
-+
-[NOTE]
-====
-As previously mentioned, the GUID here is also the Directory (tenant) ID.
-Make sure you add the trailing slash (`/`) at the end or this operation might fail.
-
-The audience parameter for access tokens is typically set with `api://` at the front.
-====
-
 
 === ID token
 
-This example shows how to configure Entra ID for authentication and authorization using ID tokens.
-
-==== Register the application
-
-. Log in to the https://portal.azure.com/#home[Azure portal].
-. Navigate to *Microsoft Entra ID > Overview*.
-. From the *Add* dropdown menu, select *App registration* and fill in the following information to create your SSO application:
-+
-image::sso-configuration-tutorials/oidc-azure-client-creation.png[title="Entra OIDC client creation"]
-The redirect URI `http://localhost:7474/browser/?idp_id=azure&auth_flow_step=redirect_uri` is the URI that will accept returned token responses after successful authentication.
-. Click *Register*.
-
+This example shows how to configure Neo4j to use an Entra ID ID token for authentication and authorization.
 
 ==== Configure Neo4j
-. After the successful app creation, on the app's *Overview* page, find the Application (client) ID value. Use it to configure the following properties in the _neo4j.conf_ file.
+
+. From the *App registrations* page, select the app you created in <<#_register_the_application,Register the application>>.
+. On the application *Overview* page, copy the Application (client) ID value and use it to configure the following properties in the _neo4j.conf_ file:
 +
 [source, properties]
 ----
-dbms.security.oidc.azure.audience=c2830ff5-86d9-4e38-8a2b-9efad6f3d06d
-dbms.security.oidc.azure.params=client_id=c2830ff5-86d9-4e38-8a2b-9efad6f3d06d;response_type=code;scope=openid profile email
+dbms.security.oidc.azure.audience=api://4376dc8b-b5af-424f-9ada-c1c1b2d416b9
```

Comment:
Is this correct to have the api:// there?

Reply:
Hmm, I am not sure now. Maybe I got confused by the note above and thought that this also applies to the ID token.

Reply:
Yeah, I don't think it should have the api://